pkg/aws/bindata.go

```diff
@@ -68,7 +68,7 @@ func (fi bindataFileInfo) Sys() interface{} {
cd\x71\x7d\xf9\xa0\x45\x67\xdf\x66\x0e\x52\x04\x64\x8b\xd1\x37\xe7\xd7\xb4\x47\x70\xea\x1e\xa7\x80\xc3\xdd\x21\xab\xbe\x17\x0a\x4e\x8a\x2f\x0a\x32\x31\xbc\x28\xd0\x54\x8b\x9d\x82\x4d\x88\x6c\x0a\x2c\x36\xfc\xc6\xb1\xc3\x48\x92\x42\x00\x56\x41\x25\x45\x05\xd5\x14\x15\x54\x54\x54\x38\x55\x51\x01\x65\x45\x85\xd4\x15\x15\x48\x58\x54\xb8\xf8\xae\x98\x02\x3c\x70\x3f\xfb\x99\x19\x4f\xf8\x06\xf6\xc2\x5e\x98\xb9\x97\x86\xd0\x5d\xed\x91\x16\x87\xec\x2c\x61\x23\xba\x56\x95\x25\xac\x95\x7b\xc5\x0a\xce\x6a\xb5\x5a\x58\x31\xd5\x51\xc1\xa5\x70\xb5\x1a\xbf\x4e\x89\x5f\xa9\xc2\x57\x3c\x35\xeb\x02\x78\xad\x2e\x71\x55\x1c\x67\xf8\x05\xa8\xc5\xea\xb2\xf1\x55\xe2\x39\x6b\xee\xc2\xd7\xc7\x5b\xa9\x57\xfc\x7a\xbb\xfb\xf1\xb5\x2b\x7a\x05\xdd\xea\x42\x33\x84\x1c\x74\x6c\xef\xca\x73\x93\x1b\x84\x0e\x37\xb4\x77\xcd\x2a\x8f\xb7\x1b\x7d\xa6\x10\x1d\xf6\x6e\x3e\xd7\x2c\x85\xf6\x78\x39\xdf\x37\xb1\x88\x1e\x13\x62\x47\x2c\xe8\xf5\x7c\x46\x5e\xab\x9c\x56\xd2\x16\x8e\xb9\xfb\x44\xa4\x61\xc7\xdc\x79\x22\xd2\xb8\x63\xee\x7e\x74\x5d\xcf\x3e\xad\xd6\x43\x16\xd4\xa0\x5f\x6f\x1b\xa2\x20\x76\xb0\xa9\x79\x6d\xe1\xda\x3b\xb7\x99\xb4\x85\x6b\x3f\x37\xe2\x43\x01\xe7\xa9\x7a\x50\x70\xf7\x63\x17\x20\x32\x1e\xe2\xee\x59\xc3\x11\xee\xda\x50\xd5\xa0\x6d\xef\xda\xe4\xd2\xa8\x6f\xcf\xb1\xc7\xdc\x4f\x1f\x60\xc0\x16\xbe\x3d\x83\xa8\x58\xf8\xf6\x0c\xaa\x62\xe1\xdb\x33\x2c\x39\xec\xdb\xbb\xd7\x15\xdc\xb7\x67\x10\x16\xdc\xb8\x67\x50\x16\xd0\xb8\x77\x2f\x2d\xa8\x73\xcf\x10\xe1\x16\x6e\x15\xb5\x9b\xe9\xb5\x67\x60\xe5\xdc\x53\xbb\x21\x3e\xfb\xb4\xda\xc2\xb9\x27\xfa\x1f\x3e\x3b\xb5\xda\xc6\xb9\x8f\xc9\x00\xd1\xb0\x75\x1f\x51\x5f\x5d\xe3\xde\x7d\x44\xd6\xa2\xb6\xf1\xee\xe3\x52\x43\x0b\xef\xde\xaa\x96\x0a\x2e\x86\xab\x55\xf9\x75\x8a\xfc\x5a\x35\x1e\x37\xef\xa9\x57\x63\x7c\xf3\x5a\x5f\xea\x42\xdd\xfb\xd8\xf4\x62\x75\xf9\x78\x10\x69\xdb\x9c\xf4\xfd\xaf\x46\x23\xb7\x69\x29\x32\x23\xab\x8f\x37\x54\x07\xf8\x51\x82\xce\xe7\xdd\x06\x19\x66\xe6\x6c\x90\xe0\x7b\x53\x5d\x37\xed\x47\x60\x64\x9d\x5d\xdb\x24\x83\x4c\x3b\xd5\x01\xdb\xcb\x75\xb7\x7e\x90\xd8\x03\x1d\x18\x76\x56\xa0\xa9\xc2\x01\x9f\x48\x77\xdc\xe3\x18\x40\xd3\xd6\x71\x2b\x6b\x98\x99\xeb\xe6\x7e\xb9\xda\x88\x6b\x88\xa5\x69\xc6\x76\x0e\x6b\x08\xa5\x09\xaa\xf3\xb7\xad\x0c\x99\x31\x92\xa6\x55\x97\x73\xd8\x30\xb3\x36\x43\x16\x6c\xa2\x6b\x84\x0e\x13\x49\x2a\x54\x28\xa9\x40\xb1\xa4\x42\x05\x93\x0a\x14\x4d\xa1\x46\xd7\x1e\x3e\x0b\x39\x7c\x11\xc6\x03\x41\x6e\x98\xd7\x90\xd2\xf9\x35\x91\x87\xcf\xa5\xc5\x5f\x50\xce\xbd\x1d\x92\xf2\x07\x1c\x2c\xfe\x80\x83\x74\x5e\x1b\x3c\xf9\x7c\x37\x8c\xd7\x0b\x48\x4f\x22\xcd\xcc\x6a\xfd\xd6\xdd\xf8\xd7\x30\x50\xc1\xf1\x08\x5e\x3e\x6e\x35\xbe\x6c\x74\x6e\x9e\xd7\xad\x86\xdf\x16\xc8\xf2\xf6\x1b\x4e\x62\xf8\x8e\xcc\x0e\xdb\x98\xae\x67\x8e\xe4\xe0\x17\x2d\x52\x1f\x4e\xee\x7d\xd9\xd0\xfe\x62\x6c\x0a\x59\xa3\xa9\x3f\xaa\x3b\xd0\xdf\xc5\x4a\xad\xa4\x91\xd8\xfa\x8c\x89\x91\xd5\x0a\x1f\x34\x3c\xd2\xf2\xf5\x9c\x61\xdf\xbc\xd6\xe6\x4b\x8c\x9c\x8c\x35\x78\xf4\x72\x11\xa5\x2f\x71\xf3\xd7\xcd\x3f\x01\x00\x00\xff\xff\x44\xe0\x9f\x7d\xc0\x8e\x00\x00")
```

Comment:
Do you know why this changed? Better compression or something?

Reply:
Not sure. Can try to generate it with the previous bindata version, otherwise I would expect the json from instances.info changed.

Reply:
It's the same bindata version as before, so that's not the case.

Reply:
@aermakov-zalando there's simply less data now:

Example of a quick diff:

```diff
   {
@@ -2368,12 +1936,8 @@
     "ebs_as_nvme": true,
     "storage": {
       "devices": 1,
-      "includes_swap_partition": false,
       "nvme_ssd": true,
-      "size": 150,
-      "ssd": true,
-      "storage_needs_initialization": false,
-      "trim_support": true
+      "storage_needs_initialization": false
     }
```

Reply:
Do we depend on any of that? :)

Reply:
Weird, I thought I regenerated bindata in https://github.com/zalando-incubator/cluster-lifecycle-manager/pull/151.